docs/croissant-spec.md

```diff
@@ -263,14 +263,40 @@ corresponding properties must be defined using `subField`.
 
 A field may have more than a single assigned `dataType`, in which case at least
 one must inform about the expected type of data (eg: `sc:Text`), while other
-types inform about the semantic being used.
+types inform about the semantic being used, possibly semantics with ML meaning.
 
 **range**:	[sc:Text](https://schema.org/Text), [sc:URL](https://schema.org/URL)
 
 **domain**:	[Field](#field)
 
+[🔗](#known-supported-data-types) Known supported data types:
 
-In the following example, the `url` field is expected to be a URL , which
+| `dataType` | Usage |
+| -------- | ----- |
+| [**`sc:Boolean`**](https://schema.org/Boolean) | Describes a boolean. |
+| [**`sc:Date`**](https://schema.org/Date) | Describes a date. |
+| [**`sc:Float`**](https://schema.org/Float) | Describes a float. |
+| [**`sc:Integer`**](https://schema.org/Integer) | Describes an integer. |
+| [**`sc:Text`**](https://schema.org/Text) | Describes a string. |
+| [**`sc:URL`**](https://schema.org/ImageObject) | Describes a URL. |
+| [**`sc:ImageObject`**](https://schema.org/ImageObject) | Describes a field containing the content of an image (pixels). |
+| [**`ml:BoundingBox`**](http://mlcommons.org/schema/BoundingBox) | Describes a bounding box. |
+| [**`sc:name`**](https://schema.org/name) | Describes a field which can be used as a human-friendly label. |
+| [**`wd:Q3985153`**](https://www.wikidata.org/wiki/Q3985153) <br/>(**Training, validation and test sets**) | Describes a field used to divide data into multiple sets according to intended usage with regards to models [training](https://mlcommons.org/definitions/training_split), [validation](https://mlcommons.org/definitions/validation_split), [testing](https://mlcommons.org/definitions/test_split), and possibly others. <br/>While any value is acceptable here, it is recommended to associate the usual splits listed above with the linked semantic URL.
+| [**`wd:Q48277`**](https://www.wikidata.org/wiki/Q48277) <br/>(**gender**) | Describes a field which values are indicative of a person gender. This can be used by Ethical AI tools to flag possible gender bias in the data. Values for this field can be associated with specific gender URLs (eg: [**`wd:Q6581097`**](https://www.wikidata.org/wiki/Q6581097), [**`wd:Q6581072`**](https://www.wikidata.org/wiki/Q6581072), etc.) |
```

Comment:
I don't think we should include Gender in this list. It doesn't really fit with the other types, and is just one particular human characteristic that may be important to understand biases in the data. Also, the wd type may not be the canonical way to describe gender information.

Instead, how about adding an "extension mechanism" section to say that we will support additional types, and maybe give this one as an example there?

Reply:
Makes sense. done.